src/live/src/user-to-user/live-service-facade.js

```diff
@@ -0,0 +1,58 @@
+import PubNub from 'pubnub';
+
+import Client from '../../../client';
+import { User } from '../../../entity';
+import { PubNubListener } from './pubnub-listener';
+import { getLiveService } from './live-service';
+
+
+function _getLiveService() {
+  const client = Client.sharedInstance();
+  return getLiveService(client);
+}
+
+function register() {
```

Comment:
I think even if we decide the public way to register would be user.register(), it should still be just a call to LiveService.register() - (as mentioned in a comment above) I feel the actual register logic for live service should be in the `LiveService` class.